### PR TITLE
BTCA-434-637 Performance data refresh and 'view in org' bug fix

### PR DESCRIPTION
**Pull request by ManasAgarwalVlocity**

### CODE REVIEW

File: lib/datapacksjob.js [hunk 1]
```diff
@@ -501,8 +501,7 @@ DataPacksJob.prototype.formatResponse = async function(jobInfo, error) {
             response.message = jobInfo.message ?  jobInfo.message : jobInfo.hasError ? jobInfo.errors.join('\n').replace(/%vlocity_namespace%/g, this.vlocity.namespace) : `${jobInfo.jobAction} Successful`;
             response.status = jobInfo.hasError ? 'error' : 'success';
         } else if (jobInfo.jobAction === 'downloadPerformanceData') {
-            response.records = jobInfo.data;
```
Comment (rutlabaga) [inline lib/datapacksjob.js:504]: Is this correct?

File: lib/datapacksjob.js [hunk 2]
```diff
@@ -501,8 +501,7 @@ DataPacksJob.prototype.formatResponse = async function(jobInfo, error) {
...
-            response.message = jobInfo.errors.length != 0 ? jobInfo.errors.join('\n') : `Success - ${jobInfo.data.length} Records`;
+            response.records = [];
```
Comment (rutlabaga) [inline lib/datapacksjob.js:504]: Is this a correct change?

Comment (rutlabaga) [reply lib/datapacksjob.js:504]: Because it loads from the file anyway in the UI?

Comment (ManasAgarwalVlocity) [reply lib/datapacksjob.js:504]: Yes this is intentional. There were basically 2 reasons for "Refresh All" getting hanged up:
1. few undefined values is userIds while fetching UserDetails for performance
2. Passing large data to UI-->  it just hangs showing job complete message and control not coming to UI
Hopefully It should fix/improve the data loading

**Pull request merged**